packages/financial-templates-lib/src/logger/SlackTransport.ts

```diff
@@ -49,18 +48,6 @@ interface SlackFormatterResponse {
 }
 // Note: info is any because it comes directly from winston.
 function slackFormatter(info: any): SlackFormatterResponse {
-  // Try and fetch injected web3 which we can use to customize the transaction receipt hyperlink:
```

Comment:
this code block is problematic. every time we send a log we are now fetching the chainId to inform logs. Originally, this was ok as we were only running bots on one chain wherein the logger could figure out what chain the tx/address was produced on and perform etherscan link formation. in the current bot context though this makes less sense as we have bots that run over multiple chains.

more over, this code block generates an error if the serverless hub is started without the `--network` flag. every execution of the hub would produce an error, as follows:

![image](https://user-images.githubusercontent.com/12886084/145974266-ceb4da1f-d31d-437e-9b8f-00ff47a6fba4.png)


removing this code has the following consequence: we no longer auto-format addresses/transactions in slack messages. IMHO this is fine as all messages that we send nowadays (relays, predominantly) are formatted from the sender using markdown, which bypasses this logic anyways.